assets/src/components/ImageHoverControls/index.js

```diff
@@ -6,21 +6,31 @@ export const ImageHoverControls = props => {
     onEdit,
     onRemove,
     isCompact,
+    isAdd,
   } = props;
 
-  return <div className="buttons-overlay">
-    <Button
+  return <div className='buttons-overlay'>
+    { isAdd && <Button
       onClick={ onEdit }
-      icon="edit"
+      icon='plus-alt2'
       isPrimary
-      className="edit-image"
+      className='edit-image'
+    >
+      { __('Add image', 'planet4-blocks-backend') }
```

Comment:
This add button is not used by the Columns block yet, because it's conditionally displayed there.
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/09068d6e833ce660389916cd296259a68b660f6c/assets/src/blocks/Columns/EditableColumns.js#L30-L39